samples/Network/deny-nsg-inbound-traffic-change/README.md

```diff
@@ -0,0 +1,34 @@
+# Denies NSG rule changes that allow all inbound traffic
+
+Denies the creation of network security group rules that allow all inbound traffic
+
+## Try on Portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/?feature.customportal=false&microsoft_azure_policy=true&microsoft_azure_policy_policyinsights=true&feature.microsoft_azure_security_policy=true&microsoft_azure_marketplace_policy=true#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FNetwork%2Fdeny-nsg-inbound-traffic-change%2Fazurepolicy.json)
+
+## Try with PowerShell
+
+````powershell
+<<<<<<< HEAD
+$definition = New-AzureRmPolicyDefinition -Name "deny-nsg-inbound-allow-all" -DisplayName "Denies NSG rule changes that allow all inbound traffic" -description "Denies people from changing NSG rules that allow all inbound traffic" -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/Network/deny-nsg-inbound-traffic-change/azurepolicy.rules.json' -Mode All
+=======
+$definition = New-AzureRmPolicyDefinition -Name "enforce-tag-value" -DisplayName "Denies NSG rule changes that allow all inbound traffic" -description "Denies the creation of network security group rules that allow all inbound traffic" -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/Network/deny-nsg-inbound-traffic-change/azurepolicy.rules.json' -Mode All
+>>>>>>> 214573a1937caf2a8d1df10531257f0a4e8a5349
```

Comment:
Looks like you ran into some git issues while merging in the local/remote changes. Please delete this file (it's in the old path), and if there's anything you think went missing let me know and we can try to recover it.